airflow/cli/commands/variable_command.py

```diff
@@ -76,7 +78,29 @@ def variables_import(args):
 
 def variables_export(args):
     """Exports all the variables to the file."""
-    _variable_export_helper(args.file)
+    var_dict = {}
+    with create_session() as session:
+        qry = session.query(Variable).all()
+
+        data = json.JSONDecoder()
+        for var in qry:
+            try:
+                val = data.decode(var.val)
+            except Exception:
+                val = var.val
+            var_dict[var.key] = val
+    file_is_stdout = _is_stdout(args.file)
+
+    with args.file as f:
+        f.write(json.dumps(var_dict, sort_keys=True, indent=4))
+    if file_is_stdout:
+        print("\nVariables successfully exported.", file=sys.stderr)
+    else:
+        print(f"Variables successfully exported to {args.file.name}.")
+
+
+def _is_stdout(fileio: io.TextIOWrapper) -> bool:
+    return fileio.name == "<stdout>"
```

Comment:
Would it be better if we treat the absence of argument `file` as an indication to use the print output to `stdout`. WDYT?